todo_app/trello.py

```diff
@@ -0,0 +1,88 @@
+import requests
+import os 
+from dotenv import load_dotenv
+from todo_app.trello_card import TrelloCard
+
+class Trello:
+
+    def __init__(self): 
+        load_dotenv()
+        self.trello_key=os.getenv('API_KEY')
+        self.trello_token=os.getenv('API_TOKEN')
+        self.board_name=os.getenv('BOARD_NAME')
+        self.memberName=os.getenv('MEMBERS_NAME')
+        self.to_do_list_name=os.getenv("TO_DO_LIST_NAME")
+        self.done_list_name=os.getenv("DONE_LIST_NAME")
+
+    def get_todo_items(self):
+        todo_items=[]
+        board_id=self.get_board_id()
+        trelloLists=self.get_lists_for_board(board_id)
+        for key in trelloLists: 
+            cards = self.get_cards_for_list(key, trelloLists[key])
+            for card_key in cards:
+                todo_items.append(cards[card_key]) 
+        return todo_items
+        
+    def add_key_and_token(self, url):   
+        return url + "key=" + self.trello_key + "&token=" + self.trello_token
+
+    def get_board_id(self):
```

Comment:
Are you saying instead of holding the board name in config I should hold the id?

Reply:
Yes I would say so. Not just because it simplifies the code but also because that's what the ID is for - uniquely and reliably identifying the entity.

- What if you have two boards with the same name?
- If you rename the board, you probably still want to keep using it, so using the ID saves you having to update the config.

Reply:
done